.github/dependabot.yaml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+    - package-ecosystem: "nuget"
+    - directory: "/"
+    - schedule:
+        interval: "daily"
```

Comment:
I've add - before the interval here as well.

Reply:
fixed